src/main/java/org/opentripplanner/middleware/controllers/api/OtpUserController.java

```diff
@@ -125,7 +140,16 @@ public VerificationResult verifyPhoneWithCode(Request req, Response res) {
                 "Unknown error encountered while checking SMS verification code"
             );
         }
-        // If the check is successful, status will be "approved"
-        return new VerificationResult(check);
+        // If the check is successful, status will be "approved".
+        VerificationResult verificationResult = new VerificationResult(check);
+        if (verificationResult.status.equals("approved")) {
+            // If the check is successful, update the OtpUser
+            // (set isPhoneNumberVerified and notificationChannel).
+            // TODO: Figure out adding a test for this update to the OtpUser record.
+            otpUser.isPhoneNumberVerified = true;
```

Comment:
Only at this point should the `otpUser.phoneNumber` be updated.

Reply:
@landonreed and @evansiroky please agree on the above comment. https://github.com/ibi-group/otp-middleware/pull/83/commits/5621a131b26f784d618e0a8cb228ba3e3700c787 does per above, https://github.com/ibi-group/otp-middleware/pull/83/commits/d88766150d215e265639e0525afce51be7c6ec66 does not.

Reply:
Using `isPhoneNumberVerified` is how we originally designed things and in my opinion it works just fine. If there needs to be further discussion or an alternative approach, that should be handled outside of this PR.

Reply:
I still disagree for reasons mentioned in company chat, but don't care enough to block progress.